packages/webpack/lib/plugins/stats.js

```diff
@@ -64,9 +64,14 @@ exports.StatsWritePlugin = class StatsWritePlugin {
             ...extractFiles(analyzeCompilation(compilation), publicPath),
           };
 
+          compiler.hooks.watchRun.tap('StatsPlugin', () => {
```

Comment:
It used to be located beside the `compiler.hooks.compilation.tap`-call, not nested inside it. Did something change?

Reply:
Oh you're right. I've changed it.